typescript/listeners/idle.py

```diff
@@ -169,7 +169,7 @@ def show_errors(self, diagno_event_body, syntactic):
     def update_status(self, view, info):
         """Update the status line with error info and quick info if no error info"""
         # Error info
-        if PHANTOM_SUPPORT:
+        if is_inline_error_tips_enabled():
```

Comment:
Is this the right check? I feel like you could get into a bad state if you

1. Enable inline error tips
2. Cause an error.
3. Disable inline error tips.

After 3, will this erase ever occur? I'm not familiar with the API, but I'm not sure it will. I think it's probably better to always erase based on `PHANTOM_SUPPORT`.